tests/baselines/reference/smartSelection_JSDocTags5.baseline

```diff
@@ -0,0 +1,43 @@
+/**
+ * @callback Foo
+ * @param {string} data
+ * @param {/**/number} [index]
```

Comment:
@andrewbranch The same case with the `@callback`
1. `number`
2. `@param {number} [index]`
3. ?

Reply:
I think you have two options for (3):

1. The whole JSDoc comment.
2. The range of all consecutive tags of the same type. So in this case, all the `@param` tags; above, all the `@property` tags. There’s similar logic for selecting all import declarations.

I think (2) would be a thoughtful touch, but I don’t think it’s strictly necessary.